pycodestyle.py

```diff
@@ -1420,7 +1420,7 @@ def stdin_get_value():
         """Read the value from stdin."""
         return TextIOWrapper(sys.stdin.buffer, errors='ignore').read()
 
-noqa = lru_cache(512)(re.compile(r'# no(?:qa|pep8)\b', re.I).search)
+noqa = lru_cache(512)(re.compile(r'# no(?:qa|pep8)(?!:)\b', re.I).search)
```

Comment:
Huh. you're right.  I'm not sure why though.  There's no `?` or `*` after the group, so I'd expect it to see >0 `:` at the end.  Sorry, I guess I haven't come across the `?! prefix` before.